### PR TITLE
TFLu: Add UInt8 support for logistic operator

### PR DESCRIPTION
**Pull request by alisonwh**
This patch is to add UInt8 support for logistic operator in TFLu.

Tried TFLu + ssd mobilenet v1 quantized model on i.mx8mp's Cortex-M7 core.

The related issue is https://github.com/tensorflow/tensorflow/issues/46882 .

### CODE REVIEW

**Comment by gbaned** [comment]
@alisonwh  Can you please sign CLA. Thanks!

**Comment by alisonwh** [comment]
> @alisonwh Can you please sign CLA. Thanks!

@gbaned Sure, I signed it.

**Comment by alisonwh** [comment]
@googlebot I signed it!

File: tensorflow/lite/kernels/internal/reference/logistic.h [hunk 1]
```diff
@@ -126,6 +126,36 @@ inline void Logistic(const RuntimeShape& input_shape, const int8_t* input_data,
   }
 }
 
+inline void Logistic(const RuntimeShape& input_shape, const uint8_t* input_data,
+                     float input_scale, int input_zero_point,
+                     const RuntimeShape& output_shape, uint8_t* output_data,
+                     float output_scale, int output_zero_point) {
+  const float cutoff_upper = 1.591576576f;
+  const float cutoff_lower = -11.091299057f;
```
Comment (miaout17) [inline tensorflow/lite/kernels/internal/reference/logistic.h:134]: This function seems almost identical to the `int8_t` version, expect the cutoff values. 
Is it possible to make the code more concise by reusing the code with a template?

**Review by miaout17 — COMMENTED** [review]
It seems this PR only contains kernel changes but no converter / quantization tool changes. Don't we need to change the code to produce models with UInt8 logistic operators?

**Comment by alisonwh** [comment]
> It seems this PR only contains kernel changes but no converter / quantization tool changes. Don't we need to change the code to produce models with UInt8 logistic operators?

I used ssd mobilenet quant model from https://tfhub.dev/tensorflow/lite-model/ssd_mobilenet_v1/1/default/1. It is using UInt8 logistic operator. So I didn't think about producing models by myself.

File: tensorflow/lite/kernels/internal/reference/logistic.h (hunk 1 above)
Comment (daverim) [reply tensorflow/lite/kernels/internal/reference/logistic.h:134]: +1

File: tensorflow/lite/micro/kernels/logistic.cc [hunk 2]
```diff
@@ -64,6 +67,11 @@ TfLiteStatus CalculateArithmeticOpData(TfLiteContext* context, TfLiteNode* node,
 
     data->input_range_radius =
         CalculateInputRadius(kInputIntegerBits, data->input_left_shift, 31);
+  } else if (input->type == kTfLiteUInt8) {
+    data->input_zero_point = input->params.zero_point;
```
Comment (daverim) [inline tensorflow/lite/micro/kernels/logistic.cc:71]: Can we do some checking here? Similarly to line 55

**Comment by gbaned** [comment]
@alisonwh  Can you please check @daverim's comments and keep us posted ? Thanks!

**Comment by gbaned** [comment]
@alisonwh  Any update on this PR? Please. Thanks!

**Review by alisonwh — COMMENTED** [review]
> @alisonwh Any update on this PR? Please. Thanks!

I am not clear about the next step. Should I create a new PR according to daverim's comments?

**Comment by gbaned** [comment]
@daverim  Can you please assist on above comments from @alisonwh. Thanks!

**Comment by gbaned** [comment]
@daverim Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
@daverim Any update on this PR? Please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Pull request closed without merging**